SLAB

increpare opened the pull request:
https://github.com/increpare/slab-markup
https://github.com/increpare/slab-markup/releases

FichteFoll commented:
You have to push a valid [semantic version](http://semver.org) [tag](http://git-scm.com/book/en/v2/Git-Basics-Tagging) (with optional leading `v`) to your repo.

You should most certainly *not* specify `source.c` as your base scope for the syntax and instead use `source.slab` (or `text.slab`).

increpare commented:
Hi, thanks for getting back - thanks for the feedback.

I'm slightly confused, because it's been so long, but these should fix things I believe/hope:

I fixed the scope.  I also added a tag to slab-markup-sublime ( https://github.com/increpare/slab-markup-sublime/releases/tag/0.2.1 )

Also, I can't remember the conventions exactly, but maybe I linked to the wrong project in my first post (I linked to the original project rather than the plugin branch) - in which case

https://github.com/increpare/slab-markup-sublime/
https://github.com/increpare/slab-markup-sublime/releases/tag/0.2.1

FichteFoll commented:
>but maybe I linked to the wrong project in my first post

You did indeed, but I noticed.
